README.md

```diff
@@ -37,6 +37,63 @@ go get github.com/aws-controllers-k8s/dev-tools/cmd/ackdev
 
 Call `ackdev help` for detailed usage instructions.
 
+### Setup
+
+To be able to use `ackdev` you'll have to run `ackdev setup` before any other command.
+
+For example
+```bash
+ackdev setup --root-directory $WORKDIR --services s3,ecr,sqs,sns
```

Comment:
I imagine most people would prefer to install ACK to their GOPATH - assuming most people who are using `ack-dev` want to actually contribute to the code. These instructions default to installing it elsewhere. IMO it would be safer to give directions to install it in the default GOPATH (leave `--root-directory` as default), and then have alternative instructions for installing it elsewhere.

Reply:
Good point!

Reply:
I was also thinking if we should auto-create `$GOPATH/src/github.com/aws-controllers-k8s` if it doesn't exist or not ..?